File: app/src/test/java/sqa_project/GeminiPro/decorator/round3/AudioSystem.java
package sqa_project.GeminiPro.decorator.round3;

public class AudioSystem extends CarDecorator {
    public AudioSystem(Car decoratedCar) {
        super(decoratedCar);
    }

    @Override
    public String getDescription() {
        return super.getDescription() + ", Audio System";
    }

    @Override
    public double getCost() {
        return super.getCost() + 500;
    }
}

File: Python/ChatGPT4o/Decorator/round1/AudioSystem.py
from CarDecorator import CarDecorator

class AudioSystem(CarDecorator):
    """Decorator class for adding an audio system to a car."""

    def description(self):
        """Adds an audio system to the car description."""
        return self._car.description() + ", Audio System"

    def cost(self):
        """Adds the cost of an audio system to the car."""
        return self._car.cost() + 1500